plugin/backend_linux.go

```diff
@@ -641,7 +641,10 @@ func (pm *Manager) Remove(name string, config *types.PluginRmConfig) error {
 
 	removeDir := pluginDir + "-removing"
```

Comment:
I like the idea, but I would suggest to first try to remove the previous `-removing` first, it shouldn't be busy by that time.